python/cudf/cudf/_libxx/includes/join.pxd

```diff
@@ -0,0 +1,28 @@
+from libcpp.vector cimport vector
+from libcpp.pair cimport pair
+from cudf._libxx.lib cimport *
+
+cdef extern from "cudf/join.hpp" namespace "cudf::experimental" nogil:
+    cdef unique_ptr[table] cpp_inner_join "cudf::experimental::inner_join" (
+        const table_view left,
+        const table_view right,
+        const vector[int] left_on,
+        const vector[int] right_on,
+        const vector[pair[int, int]] columns_in_common
+    ) except +
+cdef extern from "cudf/join.hpp" namespace "cudf::experimental" nogil:
```

Comment:
Only need to extern once and then you can declare each of these functions under the extern scope

Reply:
Fixed